File: 2167/2167.cpp
#include <iostream>
#include <vector>

int main(){
    int n, r, ans=0;
    std::vector<int> rpms;
    std::cin >> n;

    for (int i=0;i<n;i++){
        std::cin >> r;
        rpms.push_back(r);
    }

    for (int i=1;i<n;i++){
        if (rpms[i] < rpms[i-1]){
            ans = i + 1;
            break;
        }
    }

    std::cout << ans << std::endl;

    return 0;
}

File: 2167/2167.rs
use std::io;

fn input() -> String {
    let mut entrada : String = String::new();
    io::stdin()
        .read_line(&mut entrada)
        .expect("Erro ao ler a entrada!");
    entrada.to_string()
}

fn main(){
    let entrada : String = input();
    let n : usize = entrada.trim().parse().expect("msg");

    let entrada_rpms : String = input();
    let rpms : Vec<u32> = entrada_rpms
        .split_whitespace()
        .take(n)
        .map(|s| s.parse::<u32>().expect("msg"))
        .collect();
    let mut ans: usize = 0;

    for i in 1..n{
        if rpms[i] < rpms[i-1]{
            ans = i + 1;
            break;
        }
    }
    
    println!("{}", ans)
}